app/controllers/openid_connect/logout_controller.rb

```diff
@@ -61,6 +61,7 @@ def handle_successful_logout_request(result, redirect_uri)
           post_logout_redirect_uri: logout_params[:post_logout_redirect_uri],
         }
         @params[:state] = logout_params[:state] if !logout_params[:state].nil?
+        @service_provider = @logout_form.service_provider&.friendly_name
```

Comment:
Since there's a risk of mistaking this variable as being the full service provider object, could we update the name to reflect that it's the service provider's friendly name?

```suggestion
        @service_provider_name = @logout_form.service_provider&.friendly_name
```

Or, alternatively, could we change this to actually assign the full object and pick the `friendly_name` property in the view?

```suggestion
        @service_provider = @logout_form.service_provider
```

Reply:
Yep, good point. We only care about the name, so I'm going to go for the first.